CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [[v4.1.5](https://github.com/multiversx/mx-sdk-dapp/pull/1411)] - 2025-05-07
+
+- [Added support for two cross-window wallets](https://github.com/multiversx/mx-sdk-dapp/pull/1410)
```

Comment:
This is incorrect. This should be #1411. Same for the release `Pull Request`, should be a number above.